packages/teams-js/src/public/appInitialization.ts

```diff
@@ -1,9 +1,20 @@
-import { app } from './app';
+import { ApiName, ApiVersionNumber, getApiVersionTag } from '../internal/telemetry';
+import {
+  app,
+  notifyAppLoadedHelper,
+  notifyExpectedFailureHelper,
+  notifyFailureHelper,
+  notifySuccessHelper,
+} from './app';
 
 /**
  * @deprecated
  * As of 2.0.0, please use {@link app} namespace instead.
+ *
+ * v2 APIs telemetry file: All of APIs in this capability file should send out API version v2 ONLY
  */
+const appInitializationTelemetryVersionNumber: ApiVersionNumber = ApiVersionNumber.V_1;
```

Comment:
>V_1 [](http://example.com/codeflow?start=83&length=3)

How do I reconcile this being V1 with the comment above saying that it should send out V2 ONLY?

Reply:
This should be V1. Comment is incorrect. I need to fix it.

Reply:
Updated.

Reply:
Per comment on line 4-5, it is also easy to refer this is a V1 capability which started from TJS V1. The comment saying it is a V2 is a typo error.